docs/reference/kubo/cli.md

```diff
@@ -5329,7 +5329,7 @@ DESCRIPTION
   id. To specify a specific protocol, use the 'proto' option. The 'peer' and
   'proto' options cannot be specified simultaneously. The protocols that are
   queried using this method are outlined in the specification:
-  https://github.com/libp2p/specs/blob/master/7-properties.md#757-protocol-multicodecs
+  https://github.com/libp2p/specs/blob/master/_archive/7-properties.md#757-protocol-multicodecs
```

Comment:
@standstaff  this file is generate from Kubo sources, please open PR against https://github.com/ipfs/kubo/blob/642e58ef3b17784662b4758214a1a58be92af17f/core/commands/stat.go#L59 instead :pray:

Reply:
Thanks!